webapi/CopilotChat/Models/ChatMessage.cs

```diff
@@ -110,6 +110,12 @@ public enum ChatMessageType
     [JsonPropertyName("type")]
     public ChatMessageType Type { get; set; }
 
+    /// <summary>
+    /// Counts of total token usage used to generate bot response.
+    /// </summary>
+    [JsonPropertyName("tokenUsage")]
+    public TokenUsage? TokenUsage { get; set; }
```

Comment:
I think life will be probably easier if token usage cannot be null.

Reply:
This was intentional to force token usage calculation to be explicit when creating new bot response message. If it hasn't been explicitly calculated, we show an unknown state since 0 isn't truthy for all scenarios. -- sets a better standard for token management

Reply:
Desmond left a similar comment up top, so let's continue that convo up there. Resolving this convo as a dupe